token/cli/src/main.rs

```diff
@@ -2738,6 +2747,90 @@ async fn command_withdraw_withheld_tokens(
     Ok(results.join(""))
 }
 
+async fn command_update_confidential_transfer_settings(
+    config: &Config<'_>,
+    token_pubkey: Pubkey,
+    authority: Pubkey,
+    auto_approve: bool,
+    auditor_pubkey: Option<ElGamalPubkey>,
+    bulk_signers: Vec<Arc<dyn Signer>>,
+) -> CommandResult {
+    if !config.sign_only {
+        let confidential_transfer_account = config.get_account_checked(&token_pubkey).await?;
+
+        let mint_state =
+            StateWithExtensionsOwned::<Mint>::unpack(confidential_transfer_account.data)
+                .map_err(|_| format!("Could not deserialize token mint {}", token_pubkey))?;
+
+        if let Ok(confidential_transfer_mint) =
+            mint_state.get_extension::<ConfidentialTransferMint>()
+        {
+            let expected_authority = Option::<Pubkey>::from(confidential_transfer_mint.authority);
+
+            if expected_authority != Some(authority) {
+                return Err(format!(
+                    "Mint {} has confidential transfer authority {}, but {} was provided",
+                    token_pubkey,
+                    expected_authority
+                        .map(|pubkey| pubkey.to_string())
+                        .unwrap_or_else(|| "disabled".to_string()),
+                    authority
+                )
+                .into());
+            }
+        } else {
+            return Err(format!(
+                "Mint {} does not support confidential transfers",
+                token_pubkey
+            )
+            .into());
+        }
+    }
+
+    println_display(
+        config,
+        format!(
+            "Updating confidential transfer settings for {}:",
+            token_pubkey,
+        ),
+    );
+
+    println_display(
+        config,
+        format!(
+            "  approve policy set to {}",
+            if auto_approve { "auto" } else { "manual" }
+        ),
+    );
+
+    if let Some(auditor_pubkey) = auditor_pubkey {
+        println_display(
+            config,
+            format!(
+                "  auditor encryption pubkey set to {}",
+                auditor_pubkey.to_string(),
+            ),
+        );
+    } else {
+        println_display(config, format!("  auditability set to disabled",))
```

Comment:
nit:
```suggestion
        println_display(config, format!("  auditability disabled",))
```

or maybe just `auditor set to none` is fine?